src/mscorlib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -599,27 +597,33 @@ public bool Remove(TKey key)
                 int hashCode = comparer.GetHashCode(key) & 0x7FFFFFFF;
                 int bucket = hashCode % buckets.Length;
                 int last = -1;
-                for (int i = buckets[bucket]; i >= 0; last = i, i = entries[i].next)
+                int i = buckets[bucket];
+                while (i >= 0)
```

Comment:
Ok, that's what I'd have guessed, thanks.  So this change was just for consistency?